src/contexts/LocationFinderContext.tsx

```diff
@@ -93,6 +93,13 @@ export const LocationFinderProvider = <T extends object = {}>({
     // - Load more functionality.
     const [page, setPage] = useState<number>(0);
 
+    // - Get user location on first load
+    useEffect(() => {
+        navigator.geolocation.getCurrentPosition((position) => {
```

Comment:
Should execute `getCurrentPosition` once, not on every useEffect.

Reply:
UseEffect using [] will only be called once...